### DISCUSSION TITLE
Previous versions

### DISCUSSION THREAD

**Discussion by SartoXIII** [OnionUI/Onion#685] [General]
Hi there,
May I ask for Onion 4.0.3? I own a device that has firmware 20220419**** and I wanted to take advantage of the bootcmd mod but I can only find Onion 4.0.4 and 4.1-alpha6

**Comment by Aemiii91** [comment]
You can just download the boot_mod and execute via the on-device File Explorer or Terminal
https://raw.githubusercontent.com/OnionUI/Onion/v4.0.3/static/configs/.tmp_update/config/boot_mod.sh

**Reply by SartoXIII** [reply]
Thank you for the fast response
I need to execute the sh file only the first time and it's pemanent?
I'm asking because I used Onion 4.03 in the past before formatting SD card to try for some weeks MiniUI and now returning to Onion OS.
This means that the boot_mod is already applied to my device and I don't to execute it again?

**Reply by Aemiii91** [reply]
Yes, it should already be applied then.
